edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/storage/MessageStore.cs

```diff
@@ -55,31 +55,44 @@ public void SetTimeToLive(TimeSpan timeSpan)
             Events.TtlUpdated(timeSpan);
         }
 
-        public async Task AddEndpoint(string endpointId)
+        public Task AddEndpoint(string endpointId) => this.AddEndpointQueue(endpointId, RouteFactory.DefaultPriority);
+
+        public async Task AddEndpointQueue(string endpointId, uint priority)
```

Comment:
So, thinking about this a bit more - can we move this logic one level up? That way, the MessageStore just gets an endpoint, but now instead of it being "endpointId", it is "endpointId_Pri10". That way the MessageStore API doesn't need to change at all.

Reply:
talked offline, the suggested approach will likely be more flexible for when we implement metered connection management, where the messagestore would need to expose even more parameters to set.

I'm abandoning this PR and will just have the executor calculate the queue name in the next PR.